core/src/main/java/com/gentics/mesh/core/endpoint/microschema/MicroschemaCrudHandler.java

```diff
@@ -251,4 +262,35 @@ public void handleRemoveMicroschemaFromProject(InternalActionContext ac, String
 			}
 		}, () -> ac.send(NO_CONTENT));
 	}
+
+	/**
+	 * Handle the request for the project, linked to the schema with the given UUID.
+	 * 
+	 * @param ac
+	 * @param microschemaUuid
+	 */
+	public void handleGetLinkedProjects(InternalActionContext ac, String microschemaUuid) {
+		validateParameter(microschemaUuid, "microschemaUuid");
+
+		PagingParameters pagingInfo = ac.getPagingParameters();
+		ValidationUtil.validate(pagingInfo);
+
+		utils.syncTx(ac, tx -> {
+			MicroschemaDao microschemaDao = tx.microschemaDao();
+			HibMicroschema microschema = microschemaDao.loadObjectByUuid(ac, microschemaUuid, READ_PERM);
+
+			Result<HibProject> result = microschemaDao.findLinkedProjects(microschema);
```

Comment:
This will not handle permissions on the projects (so it will return all projects the microschema is assigned to, regardless of the permissions)